### PR TITLE
Add basic GitHub actions and Dependabot to master

### PR DESCRIPTION
**Pull request by byronlin13**
*Issue #, if available:*

*Description of changes:*
* Currently there's no test for this repo. I added basic Github actions to build the project.
* Successful build on my [fork](https://github.com/byronlin13/amazon-qldb-dmv-sample-java/pull/2).
* Add Dependabot which will automatically create dependency version bump PRs.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    target-branch: "master"
+    schedule:
+      interval: "daily"
```
Comment (kylepbit) [inline .github/dependabot.yml:7]: Perhaps this should be weekly just to manage the number of PRs?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy: 
+      matrix : 
+        os: [ubuntu-latest, windows-latest]
```
Comment (kylepbit) [inline .github/workflows/build.yml:14]: Do we care about any other platforms? macOS?

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        java: [8, 11]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
```
Comment (kylepbit) [inline .github/workflows/build.yml:19]: Should this use JDK11 or 8? I thought 8 was the minimum?

File: .github/dependabot.yml (hunk 1 above)
Comment (byronlin13) [reply .github/dependabot.yml:7]: The [default max pull requests](https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates#open-pull-requests-limit) for dependabot is 5, which I think is manageable.

File: .github/workflows/build.yml (hunk 3 above)
Comment (byronlin13) [reply .github/workflows/build.yml]: The name is incorrect, it should be "Set up JDK"

This step is using `${{ matrix.java }}` so it is testing against JDK 8 and 11.

File: .github/workflows/build.yml (hunk 2 above)
Comment (byronlin13) [reply .github/workflows/build.yml]: Good point, added

**Review by alpian — APPROVED** [review]

**Pull request merged**